pkg/config/config.go

```diff
@@ -30,8 +31,8 @@ func New(cs *api.OpenShiftManagedCluster) (Interface, error) {
 		return v71.New(cs), nil
 	case "v9.0":
 		return v9.New(cs), nil
-	case "v10.0":
-		return v9.New(cs), nil
```

Comment:
oops :-)

Reply:
Yeah, I have a note in the commit about fixing this on master: https://github.com/openshift/openshift-azure/commit/bf0c2a6acee0059f86ffca2a6b19e9a9a8b262c6